.github/workflows/doc-lint-test.yaml

```diff
@@ -287,8 +286,8 @@ jobs:
           for file in ${ret[@]}; do
               echo "    $file"
           done
-          if [ ${#ret[@]} -eq 0 ]; then
-              echo "FAILED: No release note changes detected"
+          if [ -z "$ret" ]; then
+              echo "FAILED: No yaml release note changes detected"
```

Comment:
![image](https://github.com/user-attachments/assets/e9e3916e-63c2-4ed3-b51a-5b1771439791)
 I'm assuming these lines are the fix for that (incorrectly passed modified YAML release note..)

Reply:
^ yep! :)

Reply:
![image](https://github.com/user-attachments/assets/24624f8d-ce6b-4f35-85ae-c86996182421)

Successfully failed!